### PR TITLE
allow use of repo token

### PR DESCRIPTION
**Pull request by evanmcc**
Services other than travis have to use a repo token instead of a .coveralls.yml file in their repo.  this change formats and sends that token if it's set in the environment.

### CODE REVIEW

**Comment by markusn** [comment]
Thanks for the PR!

To make this feature implemented more like the other things a `rebar.config` tuple `coveralls_repo_token` which can be set from `rebar.config.script`. This tuple can then be read by `rebar_coveralls.erl` and be given as an argument to `coveralls:convert_and_send_file`. This is how reading the service name and id is implemented and it would be cleaner if the repo-token is implemented in the same way.

It would also be nice with an example on how to use this in the `README.md`.

**Comment by evanmcc** [comment]
Thanks for the feedback, I'll update the PR later today.

**Comment by evanmcc** [comment]
updated: this should also close #18 when/if it lands

File: src/coveralls.erl [hunk 1]
```diff
@@ -69,38 +69,41 @@
...
+  convert_file([Filename], ServiceJobId, ServiceName, RepoToken, S);
+convert_file([[_|_]|_]=Filenames, ServiceJobId, ServiceName, RepoToken, S) ->
   ok               = lists:foreach(
                        fun(Filename) -> ok = import(S, Filename) end,
                        Filenames),
   ConvertedModules = convert_modules(S),
+
   Str              =
-    "{~n"
+    "{~n" ++ RepoToken ++
```
Comment (markusn) [inline src/coveralls.erl]: In the previous commit the string in the sent json was repo_token: Token. Now it seems to be only Token. Why was this changed?

**Comment by markusn** [comment]
Thanks for your contribution! The changes have been merged to master. Do you need a tagged version? (Uploaded to hex.pm)

**Comment by markusn** [comment]
1.4.0 available on hex.pm
